docs-2.0/3.ngql-guide/8.clauses-and-options/return.md

```diff
@@ -0,0 +1,264 @@
+# RETURN
+
+`RETURN` defines the output of an nGQL query. To return multiple fields, separate them with commas.
+
+`RETURN` can lead a clause or a statement:
+
+* A `RETURN` clause works in openCypher statements in nGQL, such as `MATCH` or `UNWIND`.
+* A `RETURN` statement works independently to output the result of an expression.
+
+## OpenCypher compatibility
+
+This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use [`YIELD`](yield.md).
```

Comment:
`This topic applies to the openCypher syntax in nGQL only. For nGQL extensions, use YIELD.`
You can delete this now.

Reply:
Not yet:

![image](https://user-images.githubusercontent.com/69501902/109916247-3cdb5580-7cee-11eb-8791-89b731eb2adf.png)